### ISSUE TITLE
how in the hell can i download mypermissions. with the zip file does not work

### ISSUE THREAD

**Issue by Arngue** [MyEssentials/MyPermissions#15]

**Comment by bit2shift** [comment]
Assuming you have `git`, `java` and `javac` available on the command line, you can build it with just 3 commands:
```
git clone --recursive https://github.com/MyEssentials/MyPermissions.git
cd MyPermissions
gradlew build
```
The build result will appear inside the `build/libs/` folder.

**Comment by Rikoyts** [comment]
Could you send me the mod mypermissions? I can not find a way to get it, thanks!

**Comment by bit2shift** [comment]
@Rikoyts no, you have to build it yourself, otherwise you're not worthy of using it.
Get the [GitHub Desktop app](https://desktop.github.com/) and use the Git shell to run the first command from the previous comment:

    git clone --recursive https://github.com/MyEssentials/MyPermissions.git

Then install the latest [JDK 8](https://adoptopenjdk.net/), open a command prompt inside the `MyPermissions` folder and run the last command:

    gradlew build

The jar will be inside the `build/libs/` folder.

**Comment by 1whohears** [comment]
I understand these mods aren't supported anymore but why y'all gotta make me waste 30 minutes fixing all of the build scripts and then compile it. This mod was a dependency for MyTown 2 and its kinda a cringe that it has released builds but this dependency doesn't.
